js/app.js

```diff
@@ -141,6 +141,37 @@ function barChart () {
   });
 }
 
+//display the result in line chart
+function lineChart () {
+  var canvas = document.getElementById('linechart');
+  var ctx = canvas.getContext('2d')
+  // modeled after the Getting Started example in the chartJS docs
+  var lineChart = new Chart(ctx, {
+    type: 'line',
+    data: {
+      labels: productName,
+      datasets: [{
+        label: 'Times shown',
+        backgroundColor: 'rgb(251, 159, 21)',
+        borderColor: 'rgba(17, 18, 17, 0.93)',
+        data: timesShown,
+      },{
+        label: 'Times clicked',
+        backgroundColor: 'rgb(2, 10, 36)',
+        borderColor: 'rgba(17, 18, 17, 0.93)',
+        data: timesClicked,
+      }]
+    },
+    options: {
+      scales: {
+        yAxes: [{
+          stacked: true
```

Comment:
I like the stacked line chart.  It's a little easier to process than the side-by-side bars in the bar chart.